bodhi/tests/server/functional/test_generic.py

```diff
@@ -101,6 +103,81 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    def test_home_counts(self):
+        user2 = User(name=u'dudemcpants')
+        self.db.flush()
+        self.db.add(user2)
```

Comment:
Oh, from what i can gather, all tests are run authenticated (with the user "guest").

This is just creating a new user in the system to add the new updates to. The page is still logged in as guest.